main/gvfs/template.py

```diff
@@ -109,6 +110,8 @@ def _goa(self):
 
     return [
         "usr/libexec/gvfs-goa*",
```

Comment:
any reason we're not install_if'ing all of these plugins?

Reply:
no idea tbh